recipe/meta.yaml

```diff
@@ -33,8 +33,9 @@ build:
   {% endif %}
   string: "{{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build }}"
 
-  {% if mpi != 'nompi' %}
   run_exports:
+    - {{ pin_compatible('fftw', max_pin='x') }}
```

Comment:
Shouldn't this be `pin_subpackage`?

Reply:
You know more than me. Probably?

Reply:
It generated this set of pins when I rebuild galsim

```
astropy >=2.0, eigen >=3.3, fftw >=3.3.8,<4.0a0, future >=0.15, libcxx >=9.0.0, lsstdesc.coord >=1.0.5, numpy >=1.14.6,<2.0a0, pybind11 >=2.2, python >=2.7,<2.8.0a0
```

Reply:
Those seem fine, but I might be missing something as usual. :)

Reply:
That rebuild also has the pin_run_as_build stuff from a rerender, so not 100% clear which thing did what. Package level variants should have higher priority IIUIC.